packages/plazo_mx/presentation/lib/features/card/pages/card_main.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+class CardMainPage extends StatelessWidget {
```

Comment:
?

Reply:
this is card page which is I haven't implemented yet